### PR TITLE
Adding Chinese Version

### PR DESCRIPTION
**Pull request by oToToT**
This game is very popular in Taiwan, also. Therefore, I've added a Chinese version here.  

Notice that, I didn't test this PR before, so feel free to modify or reject this PR.
Additionally, if there is anything wrong with the translation, please let me know and maybe I could fix it.

### CODE REVIEW

**Comment by gorisanson** [comment]
@oToToT 

Wow, thanks a lot!

Could you update a little bit since I removed "game-last-updated" and "page-last-updated" elements, and added "Update history".

Also I want to adjust the order of languages to English | Korean | Chinese. I'll do it after you commit the update. Is it OK for you?

**Comment by gorisanson** [comment]
I think the order of languages in your commit (Chinese | English | Korean) is more natural since it is in alphabetical order. I’ll keep the order.

File: src/zh/index.html [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+            <p class="margin-top">
+              <span class="thick">小提示 5:</span><br />
+              如果你想在無網路的環境下遊玩此遊戲，你可以將此頁面安裝成App。在電腦
+              上，你可以參考
+              <a
+                href="https://support.google.com/chrome/answer/9658361?hl=zh-Hant"
+                target="_blank"
+                rel="noopener"
+                >使用Chrome瀏覽器來安裝</a
+              >。 在賞機上，你可以在此頁面上點選 「加到主螢幕」。
```
Comment (david942j) [inline src/zh/index.html]: 賞機->手機

File: src/zh/index.html [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+            <div class="align-center-horizontally">
+              <img
+                class="screenshot"
+                src="../resources/assets/images/screenshot.png"
+                alt="皮卡丘打排球之遊戲截圖"
+              />
+            </div>
+            <p>
+              皮卡丘打排球是1997年時由日本開發的一款Windows遊戲。此遊戲在我小時
+              候時於韓國非常流行。我時常與朋友一同遊玩此遊戲。
```
Comment (david942j) [inline src/zh/index.html]: redundant word
小時候"時"於

File: src/zh/index.html (hunk 1 above)
Comment (oToToT) [reply src/zh/index.html]: thanks. fixed in 21ab7a8

File: src/zh/index.html (hunk 2 above)
Comment (oToToT) [reply src/zh/index.html]: thanks. fixed in 21ab7a8

**Comment by oToToT** [comment]
@gorisanson I've added the "Update history" page, pls check it.

**Comment by gorisanson** [comment]
@oToToT Thanks a lot! You translated "Update history" page, too. Awesome!

I want to add the following sentence to the update history when I release Chinese translation version tomorrow:

2020-06-28 Add Chinese translation. (Translated by @oToToT, reviewed by @david942j)

How should I write it in Chinese?

----

And I also added to the [update history](https://gorisanson.github.io/pikachu-volleyball/en/update-history/)

2020-06-27 Create this update history page. (Suggested by @e6p77Bi8CW7zRBg)

Could you please translate it too?

**Comment by oToToT** [comment]
"2020-06-28 Add Chinese translation. (Translated by @oToToT, reviewed by @david942j)"  
could be "2020-06-28 新增中文翻譯 (由 @oToToT 翻譯，並由 @david942j 查核)"

"2020-06-27 Create this update history page. (Suggested by @e6p77Bi8CW7zRBg)"  
coulde be "2020-06-27 新增此更計計錄頁面. (由 @e6p77Bi8CW7zRBg 建議)"

**Comment by gorisanson** [comment]
@oToToT Thank you!

**Comment by oToToT** [comment]
@gorisanson Sorry, I found that the above translation contain some typos.

"2020-06-27 Create this update history page. (Suggested by @e6p77Bi8CW7zRBg)"
should be "2020-06-27 新增此更新紀錄頁面. (由 @e6p77Bi8CW7zRBg 建議)"

My bad, sorry.

**Comment by gorisanson** [comment]
@oToToT No problem:) Thanks!

**Comment by gorisanson** [comment]
@oToToT I merged your PR! Thanks again!

**Comment by gorisanson** [comment]
@oToToT 

I have a question.

Screen Shot 2020-06-28 at 2 51 48 PM

The head is still in English, i.e. "Pikachu Volleyball", not translated. Is it intended or just a mistake?
If it is a mistake, I think I should translate the head "Pikachu Volleyball" to "皮卡丘打排球". Am I right?

**Comment by oToToT** [comment]
@gorisanson yes, that's a mistake, and it should be "皮卡丘打排球". Sorry for my mindlessness.

**Comment by gorisanson** [comment]
@oToToT No problem :) Thanks!

**Comment by gorisanson** [comment]
@oToToT 

Could you help me?

https://gorisanson.github.io/pikachu-volleyball/zh/

Screen Shot 2021-06-20 at 12 34 51 PM

What is a proper translation of the "play", which means "start game", at the right top corner of the image above?
Is "開始" (I got this from google translator by translating "start") a proper one? Or Is there a better one?

**Comment by oToToT** [comment]
@gorisanson Yes, I think "開始" is a proper one. Thanks for your great work!

**Comment by gorisanson** [comment]
@oToToT 

Could you help me one more time?

---

From:
Screen Shot 2021-09-20 at 10 37 14 AM

To:
Screen Shot 2021-09-20 at 10 28 35 AM

I decided to change the expression "You can play the peer-to-peer online version here" to "You can also play the peer-to-peer online version" like the images above.

---

Also I'll change
  
"If you have a Bluetooth keyboard, you can play even on your touch/mobile device." (This statement was before the controls image.)

to

"If you have a Bluetooth keyboard, you can play also on your touch/mobile devices." (This statement is to be after the controls image.)

like the images below.

From:
Screen Shot 2021-09-20 at 10 37 25 AM

To:
Screen Shot 2021-09-20 at 10 38 01 AM

---

Could you tell me proper translations?

"You can also play the peer-to-peer online version"
"If you have a Bluetooth keyboard, you can play also on your touch/mobile devices."

**Comment by oToToT** [comment]
* "You can also play the peer-to-peer online version"  → "你也可以遊玩[線上玩家對戰版本](https://gorisanson.github.io/pikachu-volleyball-p2p-online/)。"  
* "If you have a Bluetooth keyboard, you can play also on your touch/mobile devices." → "如果你有藍牙鍵盤，你也可以嘗試在觸控/行動裝置上遊玩。"

p.s. I found that I used a less formal period "." instead of the formal one "。" before, so I think it is better to change it?

Thanks again for your great work. It is really fun to play pikachu-volleyball online.

**Comment by gorisanson** [comment]
@oToToT 
Thank you very much! I'm glad to hear that you are having fun with playing Pikachu Volleyball P2P Online.

Please check the updated page: https://gorisanson.github.io/pikachu-volleyball/zh/
Did I apply your translations and your comment on p.s. just right?

**Comment by gorisanson** [comment]
@oToToT 
Could you tell me proper translation of  "Reset to default" in Chinese, please?
I updated the game so that the game options are saved and so the "Reset to default" button was introduced.

Screen Shot 2022-05-06 at 9 26 26 PM

**Comment by oToToT** [comment]
Nice feature!

I would suggest "恢復至預設值" for "Reset to default".

**Pull request merged**